xsd/src/main/jaxb/bindings.xjb

```diff
@@ -25,10 +25,26 @@
 		</schemaBindings>
 	</bindings>
 
+    <bindings schemaLocation="../difi-begrepskatalog/xsd/utvidelser/lenke.xsd" node="/xs:schema">
```

Comment:
Dette er jeg ikke så happy med. Det er litt irriterende å måtte gjenta disse 8 linjene for hver eneste XSD som legges til i fremtiden.

Så vidt jeg kunne se finnes det ingen måte å peke på _flere_ schemas og kjøre samme `nameXmlTransform` :/ Utforsket muligheten for å lage en `utvidelser.xsd` som inkluderer `lenke.xsd` (og ev. fremtidige XSDer), meen det går ikke fordi hver data-types XSD typisk vil ha sitt eget target name space.

Kanskje trenger vi ikke prefikse med `SDP`?

Lets discuss.

Reply:
Ser annoyancen. Kan vi vurdere å lage en Maven-plugin som genererer en `bindings.xjb` basert på et litt mer ekspressivt språk som muliggjør "prefix _ælt_ med SDP"?

Kødder. Synes sånne prefixes er litt digg jeg ass. Mye likt som gjentas, men er det et stort problem?

Reply:
Næh, ikke stort problem. Det blir sånn!